src/sysusers/sysusers.c

```diff
@@ -2281,6 +2361,17 @@ static int run(int argc, char *argv[]) {
         assert(!arg_image);
 #endif
 
+#if HAVE_AUDIT
+        /* Prepare to emit audit events, but only if we're operating on the host system. */
+        if (!arg_root) {
+                /* If the kernel lacks netlink or audit support, don't worry about it. */
+                c.audit_fd = audit_open();
+                if (c.audit_fd < 0)
+                        log_full_errno(IN_SET(errno, EAFNOSUPPORT, EPROTONOSUPPORT) ? LOG_DEBUG : LOG_WARNING,
+                                       errno, "Failed to connect to audit log, ignoring: %m");
+        }
+#endif
```

Comment:
the very same lines are used in src/update-utmp/update-utmp.c too, hence please too add a helper call for this that becomes a NOP if audit is compiled out